ledger/archive.gradle

```diff
@@ -0,0 +1,56 @@
+apply plugin: 'maven-publish'
+apply plugin: 'signing'
+
+publishing {
+    publications {
+        mavenJava(MavenPublication) {
+            artifactId = 'scalardl-ledger'
```

Comment:
@feeblefakie Not sure, but do we need to do something to release a new artifact in Maven?

Reply:
I don't think so, but we'll see...